File: smart-fridge-java-backend/src/main/java/css/ch/smartfridgejavabackend/user/services/UserService.java
package css.ch.smartfridgejavabackend.user.services;


import css.ch.smartfridgejavabackend.auth.dto.SignupRequest;
import css.ch.smartfridgejavabackend.user.dto.UserResponseDTO;


public interface UserService {
    UserResponseDTO createUser(SignupRequest signupRequest);

    UserResponseDTO getUserById(String userId);
}

File: smart-fridge-backend/src/main/kotlin/ch/bbzw/smartfridgebackend/user/service/UserService.kt
package ch.bbzw.smartfridgebackend.user.service

import ch.bbzw.smartfridgebackend.auth.dto.SignupRequest
import ch.bbzw.smartfridgebackend.user.dto.UserResponse

interface UserService {
    fun createUser(signupRequest: SignupRequest): UserResponse

    fun getUserById(id: String): UserResponse
}
